File: feedback/src/main/java/com/kanetik/feedback/model/DataItemAdapter.java
package com.kanetik.feedback.model;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.TextView;

import com.kanetik.feedback.R;

import java.util.ArrayList;

public class DataItemAdapter extends ArrayAdapter<ContextDataItem> {
    public DataItemAdapter(Context context, ArrayList<ContextDataItem> contextDataItems) {
        super(context, 0, contextDataItems);
    }

    @Override
    public View getView(int position, View convertView, ViewGroup parent) {
        ContextDataItem contextDataItem = getItem(position);

        if (convertView == null) {
            convertView = LayoutInflater.from(getContext()).inflate(R.layout.kanetik_feedback_data_item, parent, false);
        }

        TextView key = convertView.findViewById(R.id.key);
        TextView value = convertView.findViewById(R.id.value);

        key.setText(contextDataItem.key.replace("_", " "));
        value.setText(contextDataItem.value.toString());

        return convertView;
    }
}

File: feedback/src/main/java/com/kanetik/feedback/model/DataItemAdapter.kt
package com.kanetik.feedback.model

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import android.widget.TextView
import androidx.annotation.Keep
import com.kanetik.feedback.R
import java.util.*

@Keep
class DataItemAdapter(context: Context, contextDataItems: ArrayList<ContextDataItem>) : ArrayAdapter<ContextDataItem>(context, 0, contextDataItems) {
    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        var cvtView = convertView
        val contextDataItem = getItem(position)

        if (cvtView == null) {
            cvtView = LayoutInflater.from(context).inflate(R.layout.kanetik_feedback_data_item, parent, false)
        }

        val key = convertView?.findViewById<TextView>(R.id.key)
        key?.text = contextDataItem!!.key.replace("_", " ")

        val value = convertView?.findViewById<TextView>(R.id.value)
        value?.text = contextDataItem.getValue()

        return cvtView!!
    }
}